apps/aecore/src/aec_db.erl

```diff
@@ -158,6 +182,33 @@ get_header(Hash) ->
            Header
        end).
 
+find_block(Hash) ->
+    ?t(case mnesia:read(aec_blocks, Hash) of
+           [#aec_blocks{txs = Txs}] ->
+               [#aec_headers{value = Header}] = mnesia:read(aec_headers, Hash),
+               {value, aec_blocks:from_header_and_txs(Header, Txs)};
+           [] -> none
+       end).
+
+find_header(Hash) ->
+    case ?t(mnesia:read(aec_headers, Hash)) of
+        [#aec_headers{value = Header}] -> {value, Header};
+        [] -> none
+    end.
+
+find_chain_node(Hash) ->
+    case ?t(mnesia:read(aec_headers, Hash)) of
+        [H] -> header_to_node(H);
+        [] -> none
+    end.
+
+find_chain_node_successors(Hash) ->
```

Comment:
I tested index works after restart. Good.

I built release `make prod-build`, I created config with persisted DB (and lean16 minutes for getting initial 10 blocks faster), I started node, I used this `find_chain_node_successors` that behaved as expected. I stopped and started node, and this function returned same block (linear chain) as before.